index.html

```diff
@@ -15,7 +15,7 @@ <h3 id="time"></h3>
     var fp = new Fingerprint2();
     fp.get(function(result) {
       var d2 = new Date();
-      console.log("Time took to calculate the fingerprint: " + (d2 - d1) + "ms");
+      console.log("Time taken to calculate the fingerprint: " + (d2 - d1) + "ms");
       console.log(result);
```

Comment:
this one is not actually an error, US English is fine with past simple for recently occurred events :)